### PR TITLE
deleting old community initiative and working group

### PR DESCRIPTION
**Pull request by vinodkahuja**
Deleting community initiatives and working group folder and its subfolders as all the needed content is moved to chaoss-groups folder and is already live on the website.

Signed-off-by: Vinod K. Ahuja <vahuja@unomaha.edu>

### CODE REVIEW

File: community-initiatives-and-working-groups/community-initiatives/dei-badging/project-badging/guidelines.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
-# CHAOSS Project Badging Submission Guidelines
```
Comment (sgoggins) [inline community-initiatives-and-working-groups/community-initiatives/dei-badging/project-badging/guidelines.md:1]: What's replacing this?

**Comment by geekygirldawn** [comment]
I'm always a bit concerned about deleting files and directories, since we should always assume that other people will have linked to those pages from their blogs, websites, social media, and other posts. 

It might be better to leave at least some of those pages intact in the repo, but replace the contents with a link to where the new content can be found?

**Comment by vinodkahuja** [comment]
@sgoggins @geekygirldawn Most of this content is moved to [CHAOSS groups](https://github.com/chaoss/community/tree/main/chaoss-groups) and is reorganized following CHAOSS governance. The folder that I created PR for deleting is no longer displayed on the website.

**Comment by germonprez** [comment]
@vinodkahuja -- could you do what @geekygirldawn suggests? Just delete the content of the page and include a link to the new page? This would leave the file but a greatly reduced file (only a pointer to the new file).

**Comment by vinodkahuja** [comment]
@germonprez @geekygirldawn I can only provide pointers to the content moved to other places. In many cases, half of the content is moved, and half is removed as it is no longer applicable. In CHAOSS, our work has evolved over a period of time, and we don’t follow the same procedure that we used to follow. For example, we no longer perform some activities, like translation and MARS project. These are suggested for deletion in this PR. I cannot point to any places as these things are no longer used. 

Now, it is a design decision we have to take whether we want to keep a past history of how we used to work or keep how we currently are organized and perform things. Even if we delete these files, GitHub preserves the history. 

Further, in the KB review meeting, each individual file in this PR was reviewed by participants in those meetings, and then it was decided which file to keep and which to delete. The exact detail of decision on each file to delete, relocate, and merge is available in [Google docs here](https://docs.google.com/spreadsheets/d/1dEMY-vtiGPdM4N4poyi_zy4jKdYloWycJl23YfTzCU8/edit?pli=1#gid=0). I have followed the same and deleted or moved the content that the group decided and that was the point of reviewing, cleaning, and simplifying the KB.

**Comment by germonprez** [comment]
@vinodkahuja will include the following disclaimer on this page: 

This page is no longer part of the CHAOSS Knowledge Base. Please update any links that may be associated with this page as this page and this statement will be removed at the start of 2024.

**Pull request closed without merging**